arbnode/dataposter/storage/storage.go

```diff
@@ -30,6 +34,44 @@ type QueuedTransaction struct {
 	NextReplacement time.Time
 }
 
+type QueuedTransactionForEncoding struct {
+	FullTx          *types.Transaction
+	Data            types.DynamicFeeTx
+	Meta            []byte
+	Sent            bool
+	Created         *RlpTime `rlp:"optional"` // may be earlier than the tx was given to the tx poster
+	NextReplacement *RlpTime `rlp:"optional"`
```

Comment:
Why not just use int64 (/*int64) here instead and not drop RlpTime struct ?

Reply:
(or uint since signed int is apparently not supported either)

Reply:
Yeah there's a bit of duplication without this and I thought we might want to use RlpTime in other places too